tools/authz_generate_rules_for_roles/main.go

```diff
@@ -0,0 +1,335 @@
+/*
+Copyright 2023 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This binary is used for generating rules for user-authz roles to
+./modules/140-user-authz/docs/README.md and ./modules/140-user-authz/docs/README_RU.md.
+It inserts data between lines "<!-- start placeholder -->" and "<!-- end placeholder -->".
+It useses rendered template from /deckhouse/modules/140-user-authz/templates/cluster-roles.yaml
+Steps to use:
+  - cd tools && go generate
+  - make lint-markdown-fix
+  - check diff for ./modules/140-user-authz/docs/README.md and ./modules/140-user-authz/docs/README_RU.md files
+*/
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"sort"
+	"strings"
+
+	"github.com/iancoleman/strcase"
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	readVerbs      = []string{"get", "list", "watch"}
+	writeVerbs     = []string{"create", "delete", "deletecollection", "patch", "update"}
+	readWriteVerbs = append(readVerbs, writeVerbs...)
+
+	readVerbsString      = sliceToString(readVerbs)
```

Comment:
`readVerbsString      = sliceToString(readVerbs)`

NIT: please drop the String suffix here and following